app/services/payment_provider_customers/adyen_service.rb

```diff
@@ -117,7 +117,7 @@ def payment_link_params
         recurringProcessingModel: 'UnscheduledCardOnFile',
         expiresAt: Time.current + 69.days,
       }
-      prms[:shopperEmail] = customer.email if customer.email
+      prms[:shopperEmail] = customer.email.split(',').first if customer.email
```

Comment:
since customer.email is something that we receive from the UI, probably it would be good if we also remove all whitespaces (if any):
`prms[:shopperEmail] = customer.email.strip.split(',').first if customer.email`